src/main/java/guides/hazelcast/tomcatsessionmanager/Application.java

```diff
@@ -0,0 +1,60 @@
+package guides.hazelcast.tomcatsessionmanager;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.JoinConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.session.HazelcastSessionManager;
+import org.springframework.beans.factory.annotation.Value;
```

Comment:
This import seems unused. Can you please also check other ones if they are necessary?

Reply:
I will have to check them